src/components/Modals/TeleportModal.vue

```diff
@@ -181,5 +208,13 @@ export default defineComponent({
     margin-left: auto;
     margin-right: auto;
   }
+
+  &-slotmenu {
+    border: 1px solid black;
```

Comment:
I would remove this line. This is why the border color seems to be a little darker than the designs
![Screen Shot 2021-05-16 at 6 22 13 AM](https://user-images.githubusercontent.com/54298311/118393845-341ccf00-b60f-11eb-9fd1-ac4b2e3e7f1b.png)